.github/workflows/linter.yml

```
@@ -0,0 +1,25 @@
+name: YABA Code Linter
+
+on:
+  push:
+  pull_request:
+    branches: [ master, containerize ]
```

```suggestion
    branches: [ master ]
```
Suggest removing the branch name from the list of branches (from here and elsewhere) so that this doesn't get cluttered with dead branches. Perhaps adding a link to the PR which shows the successful run would work as a replacement?

---

In the next commit, I have removed the branch containerize. Also, you can view how the GitHub action runs for the latest commit here: [im-prakher/BETYdb-YABA/actions](https://github.com/im-prakher/BETYdb-YABA/actions).

Right now, the Pytest action fails to generate a coverage report, here's an issue for that: #71